No longer compiles with Go 1.13

Ice no longer compiles with Go 1.13.  This is important for Galène, since some distributions still use 1.13.

```
$ /usr/lib/go-1.13/bin/go test ./...
# github.com/pion/ice/v2
../../../../pkg/mod/github.com/pion/ice/v2@v2.1.4/udp_mux.go:216:22: undefined: os.ErrDeadlineExceeded
```

--- a/README.md
+++ b/README.md
@@ -63,6 +63,7 @@ Check out the **[contributing wiki](https://github.com/pion/webrtc/wiki/Contribu
 * [Antoine Baché](https://github.com/Antonito)
 * [Will Forcey](https://github.com/wawesomeNOGUI)
 * [David Zhao](https://github.com/davidzhao)
+* [Juliusz Chroboczek](https://github.com/jech)
 
 ### License
 MIT License - see [LICENSE](LICENSE) for full text

--- a/udp_mux.go
+++ b/udp_mux.go
@@ -1,7 +1,6 @@
 package ice
 
 import (
-	"errors"
 	"io"
 	"net"
 	"os"
@@ -213,7 +212,7 @@ func (m *UDPMuxDefault) connWorker() {
 		if m.IsClosed() {
 			return
 		} else if err != nil {
-			if errors.Is(err, os.ErrDeadlineExceeded) {
+			if os.IsTimeout(err) {
 				continue
 			} else if err != io.EOF {
 				logger.Errorf("could not read udp packet: %v", err)

